less/documentedit.less

```diff
@@ -602,15 +602,17 @@ input[type="radio"] {
 
 .form-group,
 .filter.orientation {
-  #ori-n, #ori-s, #ori-w, #ori-e, #ori-se, #ori-ne, #ori-sw, #ori-nw {
-    cursor: pointer;
+  path {
     transition: 0.3s ease-in-out;
 
     &.orientation-selected {
       transition: 0.3s ease-in-out;
       fill: @C2C-orange !important;
     }
   }
+  path, text {
```

Comment:
Changing the text cursor is not enough, the action must also be applied. One should also surround both path and text with a `g` tag. At least this is how I did in my branch.